docs/schema-v2.md

```diff
@@ -45,19 +69,121 @@ Example:-
-Stores two floating-point literals named `min` and `max`. `min` must be lesser than `max`.
-Example:-
+### Decoration Object
+Stores all specifications for sun and moon configuration. For optimum results, the moon texture should be a 4 wide, 2 high stacked texture.
+The Default value stores the overworld sun and moon textures and sets all enabled to true.
+
+
+**Specification**
+|     Name    |    Datatype   |                               Description                               |      Required      | Default |
+|:-----------:|:-------------:|:-----------------------------------------------------------------------:|:------------------:|:-------:|
+| `sun`       | Namespaced Id | Specifies the location of the texture to be used for rendering the sun  | :white_check_mark: |    -    |
+| `moon`      | Namespaced Id | Specifies the location of the texture to be used for rendering the moon | :white_check_mark: |    -    |
+| `showSun`   | Boolean       | Specifies whether the sun should be rendered                            |         :x:        |  `true` |
+| `showMoon`  | Boolean       | Specifies whether the moon should be rendered                           |         :x:        |  `true` |
+| `showStars` | Boolean       | Specifies whether stars should be rendered                              |         :x:        |  `true` |
+
+
+**Example**
+
 ```json
 {
-  "min": 30,
-  "max": 70.5
+  "sun": "minecraft:textures/environment/sun.png",
+  "moon": "minecraft:textures/atlas/blocks.png",
+  "showSun": true,
+  "showMoon": true,
+  "showStars": true
 }
 ```
 
-### Textures Object
-Stores six identifiers of textures for each cardinal direction (north, west, south, east, up, down)
-Example:-
+
+### Height Entry Object
+Specifies a minimum and maximum height. All fields are required and must be within 0 and 256. 
+
+
+**Specification**
+
+|  Name |    Datatype    |          Description         |
+|:-----:|:--------------:|:----------------------------:|
+| `min` | Floating point | Specifies the minimum height |
+| `max` | Floating point | Specifies the maximum height |
+
+
+**Examples**
+
+```json
+{
+  "min": 60.0,
+  "max": 120.5
+}
+```
+
+### Float Vector
+Specifies a list of three floating-point literals.
+
+**Specification**
+Does not contain any fields. 
+
+**Examples**
+```json
+[
+  0.0,
+  1.0,
+  0.5
+]
+```
+
+### Rotation Object
+Specifies static and axis rotation for a skybox. 
+
+
+**Specification** 
+
+|   Name   |   Datatype   |          Description          |      Required      | Default value |
+|:--------:|:------------:|:-----------------------------:|:------------------:|:-------------:|
+| `static` | Float Vector | Specifies the static rotation | :white_check:mark: |       -       |
+| `axis`   | Float Vector | Specifies the axis rotation   |         :x:        |    [0,0,0]    |
+
+
+**Example**
+```json
+{
+  "static": [
+    0.0,
+    0.9,
+    0.4
+  ],
+  "axis": [
+    0.5,
+    0.5,
+    0.5
+  ]
+}
+```
+
+### Weather
```

Comment:
Weather should be part of the conditionals object, no?

Reply:
It is

Reply:
Oh, I see now what you did.  Looks good now that I am looking at it as a not-git diff